app/vlinsert/insertutil/common_params.go

```diff
@@ -86,9 +89,11 @@ func GetCommonParams(r *http.Request) (*CommonParams, error) {
 		IgnoreFields:     ignoreFields,
 		DecolorizeFields: decolorizeFields,
 		ExtraFields:      extraFields,
-		Debug:            debug,
-		DebugRequestURI:  debugRequestURI,
-		DebugRemoteAddr:  debugRemoteAddr,
+
+		IsDefaultTimeField: isDefaultTimeField,
```

Comment:
WDYT of using name`IsTimeFieldSet` with inverse logic? It has default value of false and doesn't require explicit initialization. 

 E.g. :

```go
if !cp.IsTimeFieldSet {
		cp.TimeFields = []string{*journaldTimeField}
	}
```